iphone/Maps/Classes/CustomAlert/Toast/Toast.swift

```diff
@@ -90,9 +101,7 @@ final class Toast: NSObject {
       UIView.animate(withDuration: kDefaultAnimationDuration,
                      animations: { self.blurView.alpha = 0 }) { [self] _ in self.blurView.removeFromSuperview() }
     }
-  }
-
-  @objc private func onTimer() {
-    hide()
+    
+    Self.toasts.removeAll(where: { $0 === self })
```

Comment:
I think it's better to try to place this line into the `UIView.animate`'s completion handler to remove the toast instance as a final step.